operator/src/network/controller.rs

```diff
@@ -1706,21 +1702,146 @@ mod tests {
+                ready_replicas: 0,
+                namespace: Some("keramik-test".to_owned()),
+                peers: vec![],
+                ..Default::default()
+            });
+        let mock_rpc_client = default_ipfs_rpc_mock();
+        let mut stub = Stub::default().with_network(network.clone());
+        stub.status.patch(expect![[r#"
+            --- original
+            +++ modified
+            @@ -9,7 +9,7 @@
+                   "status": {
+                     "replicas": 0,
+                     "readyReplicas": 0,
+            -        "namespace": null,
+            +        "namespace": "keramik-test",
+                     "peers": []
+                   }
+                 },
+        "#]]);
+        stub.ceramic_configmaps
+            .push(expect_file!["./testdata/go_ipfs_configmap_commands"].into());
+        stub.ceramic_stateful_set.patch(expect![[r#"
+            --- original
+            +++ modified
+            @@ -137,39 +137,13 @@
+                             ]
+                           },
+                           {
+            -                "env": [
+            -                  {
+            -                    "name": "RUST_LOG",
+            -                    "value": "info,ceramic_one=debug,tracing_actix_web=debug"
+            -                  },
+            -                  {
+            -                    "name": "CERAMIC_ONE_BIND_ADDRESS",
+            -                    "value": "0.0.0.0:5001"
+            -                  },
+            -                  {
+            -                    "name": "CERAMIC_ONE_METRICS",
+            -                    "value": "true"
+            -                  },
+            -                  {
+            -                    "name": "CERAMIC_ONE_METRICS_BIND_ADDRESS",
+            -                    "value": "0.0.0.0:9090"
+            -                  },
+            -                  {
+            -                    "name": "CERAMIC_ONE_SWARM_ADDRESSES",
+            -                    "value": "/ip4/0.0.0.0/tcp/4001"
+            -                  },
+            -                  {
+            -                    "name": "CERAMIC_ONE_STORE_DIR",
+            -                    "value": "/data/ipfs"
+            -                  }
+            -                ],
+            -                "image": "public.ecr.aws/r5b3e0r5/3box/ceramic-one:latest",
+            -                "imagePullPolicy": "Always",
+            +                "image": "ipfs/kubo:v0.19.1@sha256:c4527752a2130f55090be89ade8dde8f8a5328ec72570676b90f66e2cabf827d",
+            +                "imagePullPolicy": "IfNotPresent",
+                             "name": "ipfs",
+                             "ports": [
+                               {
+                                 "containerPort": 4001,
+            -                    "name": "swarm-tcp",
+            +                    "name": "swarm",
```

Comment:
not part of your change but we should use the same name for both Go and Rust flavors of the swarm port. `swarm-tcp` is the better name IMO but happy with either as long as they are consistent.